src/clr-icons/shapes/essential.umd.ts

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2016-2018 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+import {
+  ClrShapeBanSVG,
+  ClrShapeBoltSVG,
+  ClrShapeBubbleExclamationSVG,
+  ClrShapeFileSVG,
+  ClrShapeLoginSVG,
+  ClrShapeLogoutSVG,
+  ClrShapeNoteSVG,
+  ClrShapeOrganizationSVG,
+  ClrShapePaperclipSVG,
+  ClrShapePencilSVG,
+  ClrShapePinboardSVG,
+  ClrShapePlusSVG,
+  ClrShapeResizeSVG,
+  ClrShapeShrinkSVG,
+  ClrShapeTimesCircleSVG,
+  ClrShapeUsersSVG,
+  EssentialShapes,
+} from './essential';
+import { descriptorConfig } from '../utils/descriptor-config';
+
+export * from './essential';
+
+Object.defineProperty(EssentialShapes, 'edit', descriptorConfig(ClrShapePencilSVG));
```

Comment:
I believe we can remove all of these aliases from the umd files right?

Reply:
Yeah, right. It looks like we already created the aliases and assigned them to `EssentialShapes ` in the `essential`.ts file.

Reply:
Yes. Just confirmed and will remove.